src/ontology/lexmatch/mondo-only/unmapped_doid_mondo.tsv

```diff
@@ -1,35 +1,35 @@
-MONDO:0002891	obsolete gastrointestinal neuroendocrine benign tumor	DOID:4148	MONDO:equivalentTo	gastrointestinal neuroendocrine benign tumor	semapv:UnspecifiedMatching					
-MONDO:0003043	obsolete extraskeletal mesenchymal chondrosarcoma	DOID:4548	MONDO:equivalentTo	extraskeletal mesenchymal chondrosarcoma	semapv:UnspecifiedMatching					
-MONDO:0003456	obsolete bile duct mucinous cystic neoplasm	DOID:5469	MONDO:equivalentTo	biliary tract intraductal papillary mucinous neoplasm	semapv:UnspecifiedMatching					
-MONDO:0003804	obsolete blood protein disease	DOID:620	MONDO:equivalentTo	blood protein disease	semapv:UnspecifiedMatching					
-MONDO:0004167	obsolete lung clear cell carcinoma	DOID:7267	MONDO:equivalentTo	lung clear cell carcinoma	semapv:UnspecifiedMatching					
-MONDO:0004268	obsolete subareolar duct papillomatosis	DOID:7533	MONDO:equivalentTo	subareolar duct papillomatosis	semapv:UnspecifiedMatching					
-MONDO:0004391	obsolete adult extraosseous chondrosarcoma	DOID:7902	MONDO:equivalentTo	adult extraskeletal myxoid chondrosarcoma	semapv:UnspecifiedMatching					
-MONDO:0004398	mediastinal schwannoma	DOID:7922	MONDO:equivalentTo	obsolete benign mediastinal neurilemmoma	semapv:UnspecifiedMatching					
-MONDO:0005128	obsolete sensory system disease	DOID:0050155	MONDO:equivalentTo	sensory system disease	semapv:UnspecifiedMatching					
-MONDO:0005862	obsolete multiple chemical sensitivity	DOID:4661	MONDO:equivalentTo	multiple chemical sensitivity	semapv:UnspecifiedMatching					
-MONDO:0012144	Waardenburg syndrome type 2D	DOID:0110952	MONDO:equivalentTo	obsolete Waardenburg syndrome type 2D	semapv:UnspecifiedMatching					
-MONDO:0017841	obsolete autoimmune disease with skin involvement	DOID:0060039	MONDO:equivalentTo	autoimmune disease of skin and connective tissue	semapv:UnspecifiedMatching					
-MONDO:0018251	obsolete glycogen storage disease due to phosphorylase kinase deficiency	DOID:0050594	MONDO:equivalentTo	glycogen storage disease IX	semapv:UnspecifiedMatching					
-MONDO:0021199	obsolete disease by anatomical system	DOID:7	MONDO:equivalentTo	disease of anatomical entity	semapv:UnspecifiedMatching					
-MONDO:0021668	obsolete disorder involving pain	DOID:0060164	MONDO:equivalentTo	pain disorder	semapv:UnspecifiedMatching					
-MONDO:0024657	obsolete macrocystic neurilemmoma	DOID:3203	MONDO:equivalentTo	macrocystic neurilemmoma	semapv:UnspecifiedMatching					
-MONDO:0100140	obsolete mild COVID-19 infection	DOID:0081014	MONDO:equivalentTo	non-severe COVID-19	semapv:UnspecifiedMatching					
-MONDO:0100142	obsolete severe COVID-19 infection	DOID:0081013	MONDO:equivalentTo	severe COVID-19	semapv:UnspecifiedMatching					
-MONDO:0100143	obsolete critical COVID-19 infection	DOID:0081012	MONDO:equivalentTo	critical COVID-19	semapv:UnspecifiedMatching					
+MONDO:0000275	obsolete monogenic disease	DOID:0050177	MONDO:equivalentTo	monogenic disease	semapv:UnspecifiedMatching		0.5			
```

Comment:
@twhetzel @joeflack4 Here the decision of applying the default confidence seems to manifest

Reply:
So, for these `unmapped` outputs only, I take it, all rows will now have a confidence of `0.5` rather than null/nan.